scripts-base/src/initializeComponents.ts

```diff
@@ -27,9 +27,7 @@ export const initializeComponents = (
 			const Component = componentsByName[component.name] // class
 
 			const placement =
-				typeof component.place === 'string'
-					? document.querySelector(component.place)
-					: component.place || document.body
+				typeof component.place === 'string' ? document.querySelector(component.place) : component.place || document.body
```

Comment:
Přemýšlím o výchozí hodnotě na `body`, zdali spíše nezměnit na `document`. Přijde mi to jaké více používané umístění, pokud nemám instanci přímo na elementu.

Reply:
To se podle mě neděje, alespoň já jsem se setkal s mnoha a mnoha komponentami na `body`, ale žádné na `document`. Už jenom kvůli tomu, ale také samozřejmě obecně, podle mě absolutně není dobré zanášet tak velký BC break.

Reply:
Souhlasím s Jirkou.

Reply:
Máte někde příklady, kde nepoužíváte `place` prosím? Z mé zkušenosti když neinicializuji přímo na konkrétním HTML elementu (což je i `body`), tak se používá `document` jako nejvyšší rodič. Např. delegované eventy se standardně zachytávají až na `document`.

Jinak BC break argument tady není relevantní (protože toto celé je BC break).

Reply:
Třeba `Shape`, a to na úplně všech projektech.

A tady žádný BC break není.

Reply:
✔️ Shape